client/my-sites/earn/memberships/index.jsx

```diff
@@ -595,11 +597,11 @@ class MembershipsSection extends Component {
 						) }
 					</div>
 					<div>
-						<h3>{ translate( 'No membership fees' ) }</h3>
-						<p>{ preventWidows( translate( 'No monthly or annual fees charged.' ) ) }</p>
+						<h3>{ translate( 'Simple fees structure' ) }</h3>
 						<p>
-							<CommissionFees commission={ commission } />
+							<CommissionFees commission={ commission } siteSlug={ siteSlug } />
 						</p>
+						<p>{ preventWidows( translate( 'No fixed monthly or annual fees charged.' ) ) }</p>
```

Comment:
@crisbusquets any copy suggestions here? (Ignore the tags in screenshot)

<img width="1156" alt="Screenshot 2023-06-02 at 16 19 46" src="https://github.com/Automattic/wp-calypso/assets/87168/ae1d0e51-2658-4131-94d4-580d09289a73">

Reply:
I like that you inverted the order: first the fees, using the same copy added on /earn, and then "No fixed monthly or annual fees charged". Looks good to me.

Reply:
@crisbusquets Thanks!

Fresh screenshots:

<img width="1079" alt="Screenshot 2023-06-05 at 22 47 19" src="https://github.com/Automattic/wp-calypso/assets/87168/114e5f32-c39e-4184-87cb-8079b4d74f3a">
<img width="1077" alt="Screenshot 2023-06-05 at 22 46 20" src="https://github.com/Automattic/wp-calypso/assets/87168/96e6deed-93f7-46ea-bdf0-efa9f82de5bc">
<img width="552" alt="Screenshot 2023-06-05 at 22 46 08" src="https://github.com/Automattic/wp-calypso/assets/87168/c8e376f9-088d-4512-a65e-c4ce67c81b75">

For your review. A bit clunky with two links but think this is good enough for now, and you'll include something better for the redesign?

Reply:
Works for me, @simison. Thanks for sharing :)